### ISSUE TITLE
Crash without WAILA installed

### ISSUE THREAD

**Issue by tlitookilakin** [Necr0/ConvenientAdditions#2]
Please make WAILA support optional rather than required. 
[crash-2016-12-03_10.58.15-client.txt](https://github.com/Necr0/ConvenientAdditions/files/628937/crash-2016-12-03_10.58.15-client.txt)
Not everyone plays using WAILA.

**Comment by QbDesu** [comment]
It's weird that this bug even occurred.  I actually don't even have WAILA installed in my own instance and it's not even a dependency. I don't and the error it threw doesn't even make any sense since the class that was "not found" should be compiled with the mod, maybe some interference with some mod other with outdated WAILA api.

Anyways, sorry to say this but I don't support "old" build. The build you use unfortunately had a lot of bugs. If you still experience the bug in the latest build which shouldn't be the case, as I debug without having WAILA installed myself, make sure to report it. This also means I don't support 1.10.2 anymore, which I'm sorry for. If you want to stick to 1.10.2 you unfortunately cannot use Convenient Additions as a lot of critical bugs have been resolved in the last updates.

I'm sorry if that means I wasted your time with my mod but I do not have the time nor the motivation to maintain the mod for 2 versions at once. As it currently stands I will keep my hands off compatibility layers for now. Sorry 😞
